src/jsonconfig.h

```diff
@@ -101,16 +106,28 @@ class Config{
         else{
             use_pyro = true;
         }
+        
+        if(doc.find("spawn") != doc.end()){
+            std::vector<double> spawn_location = doc.at("spawn");
+            assert(spawn_location.size()==3);
+            spawn = geo::Pose3D(spawn_location[0],spawn_location[1],0,0,0,spawn_location[2]); // config value is [x,y,theta] = [x,y,0,0,0,yaw]
+        }
+        else{
+            spawn = geo::Pose3D::identity(); // [x,y,theta] = [0,0,0]
+        }
+        
 
     }
 
-    void print(){
+    void print(){// todo add custom spawn location
         std::cout << "Simulator configuration settings:" << std::endl;
         std::cout << "Uncertain odom: " << uncertain_odom.value() << std::endl;
         std::cout << "Show full map: " << show_full_map.value() << std::endl;
         std::cout << "disable_speedcap: " << disable_speedcap.value() << std::endl;
         std::cout << "enable taco: " << enable_taco.value() << std::endl;
         std::cout << "use pyro: " << use_pyro.value() << std::endl;
+        std::cout << "Spawn Location: "<< spawn.value().getOrigin()<< std::endl;
+        std::cout << "Spawn Rotation: " << spawn.value().getBasis() << std::endl; // Todo get yaw, geolib implm gives unexpected results.
```

Comment:
I think geolib rotations have a getRPY function for this.

Reply:
Correct. There is a getYaw function to get the yaw from a Pose3D. This function, however, doesn't behave as I would expect. I've made an issue in geolib, to investigate what happens on that side. 

https://github.com/tue-robotics/geolib2/issues/62#issue-1247648600